### ISSUE TITLE
Woodpecker `2.7.2` broke login auth redirects, always returns blank page.

### ISSUE THREAD

**Issue by dessalines** [woodpecker-ci/woodpecker#4294]
### Component

server

### Describe the bug

With the `2.7.1` -> `2.7.2` upgrade, woodpecker always returns a blank page. It seems to be trying to redirect to the `/login?url=/repos` page, but returns a blank page

### Steps to reproduce

1. Have a working 2.7.1 install
2. Upgrade to 2.7.2
3. All pages are blank, and the loging redirect fails.

### Expected behavior

It should correctly handle login redirects

### System Info

```shell
`"source":"https://github.com/woodpecker-ci/woodpecker","version":"2.7.2"}`
```

Luckily there were no migrations as far as I can tell, so switching back to the `v2.7.1` docker tag seems to be working fine.

### Additional context

NA

### Validations

- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]

**Comment by zc-devs** [comment]
Perhaps, related #4194

**Comment by devlux** [comment]
same here

**Comment by sinlov** [comment]
it look like `https://woodpecker.foo/api/stream/events`  api error

```
strict-origin-when-cross-origin
```
- Browser console error

```
index-KaW5c5Z3.js:14 SyntaxError (at index-KaW5c5Z3.js:30:2267)
    at Ja (index-KaW5c5Z3.js:30:2267)
    at bt (index-KaW5c5Z3.js:40:798)
    at ue (index-KaW5c5Z3.js:40:4679)
    at Object.pe [as t] (index-KaW5c5Z3.js:40:4736)
    at setup (Login-RBNwZx3k.js:1:1138)
    at ca (index-KaW5c5Z3.js:14:38)
    at _p (index-KaW5c5Z3.js:14:38755)
    at pp (index-KaW5c5Z3.js:14:38538)
    at K (index-KaW5c5Z3.js:14:22424)
    at B (index-KaW5c5Z3.js:14:22316)
```

**Comment by fabienengels** [comment]
Got the same issue using Caddy as reverse proxy

**Comment by romain-rossi** [comment]
Same using Traefik as reverse proxy

**Comment by pat-s** [comment]
As @zc-devs stated, it likely due to #4194, which required a dep update (#4197) to get fixed. This wasn't backported to 2.7.2, so this release is broken and unusable.

**Comment by adamcstephens** [comment]
Did you try the new release, 2.7.3? Because the linked PR was included in that.

**Comment by gramakri** [comment]
We updated the Cloudron woodpecker package. Can confirm the issue is fixed with 2.7.3
